packages/cli/src/config.ts

```diff
@@ -59,3 +61,10 @@ export const PROJECT_TEMPALTES = [
   { value: "netlify-edge-functions" as const, label: "Netlify Edge Functions" },
   { value: "ssg" as const, label: "Static Site Generation (SSG)" },
 ];
+
+// We must ensure the validated template type always matches the CLI-supported templates.
+// This is crucial for security, as template names can be used in CLI/bash environments.
+// A TypeScript failure here means the Templates type is not consistent with the CLI templates.
+expectType<TypeEqual<(typeof PROJECT_TEMPALTES)[number]["value"], Templates>>(
```

Comment:
can use just use satisfies in PROJECT_TEMPALTES?

Reply:
No, I need exact type comparison

Reply:
satisfies checks only that type extends 
i.e.
`const x= 'xx' satisfies 'hello' | 'xx'` no issues